src/client/search/SearchActions.js

```diff
@@ -22,3 +25,24 @@ const addQueryParamTypes = (searchQuery, query, paramTypes) => {
     paramTypes,
   };
 };
+
+export const INVALIDATE_FEED_RESULTS = 'INVALIDATE_FEED_RESULTS';
+export const invalidateFeedResults = () => {
+  return (dispatch, getState) => {
+    dispatch({ 'type': INVALIDATE_FEED_RESULTS });
+    dispatch(searchApiForFeed(getState().searchTerms));
+  };
+};
+
+export const UPDATE_FEED_RESULTS = 'UPDATE_FEED_RESULTS';
```

Comment:
Invalidate isn't captured or reduced anywhere right now, we'd catch it later but I can take it out from this commit as there's no clear use for it.

I'm happy to change it from UPDATE_FEED_RESULTS to RECEIVE_FEED_RESULTS